config/host-operator-config.yaml

```diff
@@ -5,4 +5,7 @@ metadata:
   namespace: ${NAMESPACE}
 type: Opaque
 data:
-  registration.service.url: ${REGISTRATION_SERVICE_URL}
\ No newline at end of file
+  registration.service.url: ${REGISTRATION_SERVICE_URL}
+  console.namespace: ${CONSOLE_NAMESPACE)
```

Comment:
@alexeykazakov shouldn't we also make the console `name` configurable? it's currently hardcoded to "console" https://github.com/codeready-toolchain/host-operator/blob/master/pkg/controller/masteruserrecord/sync.go#L150

Reply:
added here: 51c7e2524f4f8cfcb2d7c347f4b9541a579fda83